api/src/pcapi/serialization/decorator.py

```diff
@@ -132,11 +140,20 @@ def sync_validate(*args: Any, **kwargs: Any) -> Response:
             try:
                 body_params = request.get_json()
             except BadRequest as error:
-                logger.info(
-                    "Error when decoding request body: %s",
-                    error,
-                    extra={"contentTypeHeader": request.headers.get("Content-Type"), "path": request.path},
-                )
+                try:
+                    content_length = request.headers.get("Content-Length", 1000, type=int)
```

Comment:
https://flask.palletsprojects.com/en/2.2.x/api/#flask.Request.get_data
It is advised to limit request data since a client could send "dozens of megabytes"